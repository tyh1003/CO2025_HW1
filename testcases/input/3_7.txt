9
78 3 93 32 11 98 72 99 76
