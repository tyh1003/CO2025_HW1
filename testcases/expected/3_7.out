3 11 32 72 76 78 93 98 99 
